app/src/components/user/AccountIndexPage/ManageAccount/ActiveLicenseInfo/index.js

```diff
@@ -69,10 +79,10 @@ const ActiveLicenseInfo = ({
           <SubscriptionInfo
             hideShadow={hideShadow}
             subscriptionDetails={{
-              validFrom: new Date("2023-08-16T00:00:00Z").getTime(),
+              validFrom: new Date(sessionReplayLifeTimeDetails.date).getTime(),
```

Comment:
This will break for deals claimed from producthunt. Backward compatibility missing as this is in `epoch` not in dict in case of PH deal